Library/Homebrew/cmd/update.rb

```diff
@@ -251,7 +252,7 @@ def dump
     # Key Legend: Added (A), Copied (C), Deleted (D), Modified (M), Renamed (R)
 
     dump_formula_report :A, "New Formulae"
-    dump_formula_report :M, "Updated Formulae"
+    dump_formula_report :M, "Upgraded Formulae"
```

Comment:
Sorry to bikeshed but I like `Updated` here to avoid implying that `update` has any functional overlap with `upgrade`.